frontend/src/scenes/settings/Settings.tsx

```diff
@@ -143,7 +143,7 @@ export function Settings({
             <AuthenticationAreaComponent>
                 <div className="flex-1 w-full gap-y-2 min-w-0">
```

Comment:
This class has gap-y-2 which if it has `flex flex-col` would work. Or can be swapped to the `space-y-2` instead which would be nicer... i dunno not important

Reply:
Good call!